.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "rs-client-libraries"]
```

Comment:
I've removed the git submodule because rs-server doesn't use rs-client. I've added some code that is duplicated between rs-server and rs-client but we'll decide later with Vincent if we put it somewhere in common.